packages/main/src/plugin/configuration-registry.ts

```diff
@@ -42,6 +42,12 @@ export interface IConfigurationChangeEvent {
   scope: containerDesktopAPI.ConfigurationScope;
 }
 
+export interface IProviderContainerConfigurationPropertyRecordedSchema extends IConfigurationPropertyRecordedSchema {
+  value?: any;
+  container: string;
+  providerId: string;
+}
+
```

Comment:
Could you explain why we need this schema ?
The schema declare the definition, not the values